lua/telescope/utils.lua

```diff
@@ -295,4 +298,46 @@ utils.align_str = function(string, width, right_justify)
     or string..string.rep(" ", width - str_len)
 end
 
+if has_devicons then
+  if not devicons.has_loaded() then
+   devicons.setup()
+  end
+
+  utils.transform_devicons = function(filename, display, disable_devicons)
```

Comment:
I would do this differently and i think that is the issues:
```lua
utils.transform_devicons = (function()
  if has_devicons then
    -- load devicons
    return function(filename, display, disable_devicons)
      -- the content of transform_devicons
    end
  else
    return function(_, display, _)
      return display
    end
  end
end)()
```

And then the same for `get_devicons`

The function will than run the first time and will initialize `transform_devicons` and `get_devicons`

Does this make sense for you?